l10n_br_account/models/fiscal_document.py

```diff
@@ -26,3 +26,9 @@ def unlink(self):
                 _("You cannot delete a fiscal document " "which is not draft state.")
             )
         return super().unlink()
+
+    @api.model
+    def create(self, values):
+        if values.get("fiscal_line_ids") and values.get("active") is not False:
+            values.update({"fiscal_line_ids": False})
```

Comment:
no geral tou OK, mas tem duas coisas:

1. eu acho que merece uma linha de comentário no código para a gente lembrar do porque disso.
2. já que vc vai introduzir esse override, eu acho bom já dar uma limpa na assinatura do método create: na v14, no models.py o create tem o decorator @api.model_create_multi e o argumento é um vals_list no qual vc deve fazer um loop. Ambos assinaturas do do create funcionam porque o decorador enfia o vals num [] caso vals for um Mapping. Mas enfim é melhor ser explicito e permitir a criação de múltiplos registros de uma vez como o ORM permite. Ai ja que seria para acertar aqui, deveria aproveitar e acertar no super, ou seja aqui tb https://github.com/OCA/l10n-brazil/blob/14.0/l10n_br_fiscal/models/document.py#L364 Tem outros create que podem ser transformados tb, mas pode ser feito em outros PRs.

O que vc acha?

Reply:
feito..